pennylane/templates/embeddings/angle.py

```diff
@@ -73,7 +73,7 @@ def circuit(feature_vector):
     num_wires = AnyWires
     grad_method = None
 
-    def __init__(self, features, wires, rotation="X", do_queue=True, id=None):
+    def __init__(self, features, wires, rotation="X", do_queue=None, id=None):
```

Comment:
Could add `do_queue` and `id` to the "Args" section above.

Reply:
94915cd8a1748832d3d6d4ef3b35423e7b8ecb0b